src/LCT.PackageIdentifier/BomCreator.cs

```diff
@@ -101,30 +101,30 @@ private static void WriteContentToCycloneDxBOM(CommonAppSettings appSettings, Bo
         private async Task<Bom> CallPackageParser(CommonAppSettings appSettings)
         {
             IParser parser;
-
+            ICycloneDXBomParser cycloneDXBomParser = new CycloneDXBomParser();
```

Comment:
can we avoid this , creation of object inside the method?

Reply:
solved